config.json

```diff
@@ -496,6 +496,11 @@
       "difficulty": 1,
       "slug": "forth",
       "topics": []
+    },
+    {
+      "difficulty": 1,
+      "slug": "change",
+      "topics": []
```

Comment:
I'm thinking for the "change" exercise, these topics might apply:  Control-flow (loops), Arrays, and Algorithms.   What do you think ?